administrator/components/com_admin/sql/updates/mysql/4.1.5-2022-06-02.sql

```diff
@@ -0,0 +1,11 @@
+UPDATE `#__extensions`
+   SET `params` = REPLACE(`params`,'"negotiate_tls":1','"encryption":"tls"')
+ WHERE `client_id` = 0
+   AND `name` = 'plg_authentication_ldap'
+   AND `params` LIKE '{%"negotiate_tls":1%}';
+
+UPDATE `#__extensions`
+   SET `params` = REPLACE(`params`,'"negotiate_tls":0','"encryption":"none"')
+ WHERE `client_id` = 0
+   AND `name` = 'plg_authentication_ldap'
+   AND `params` LIKE '{%"negotiate_tls":0%}';
```

Comment:
```suggestion
UPDATE `#__extensions`
   SET `params` = REPLACE(`params`, '"negotiate_tls":1', '"encryption":"tls"')
 WHERE `name` = 'plg_authentication_ldap'
   AND `type` = 'plugin'
   AND `element` = 'plugin'
   AND `folder` = 'authentication'
   AND `client_id` = 0
   AND `params` LIKE '{%"negotiate_tls":1%}';

UPDATE `#__extensions`
   SET `params` = REPLACE(`params`, '"negotiate_tls":0', '"encryption":"none"')
 WHERE `name` = 'plg_authentication_ldap'
   AND `type` = 'plugin'
   AND `element` = 'plugin'
   AND `folder` = 'authentication'
   AND `client_id` = 0
   AND `params` LIKE '{%"negotiate_tls":0%}';

```
More precise WHERE clause and a bit code style.